.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -15,6 +15,13 @@ assignees: ''
 ### steps to reproduce
  - steps
```

Comment:
Consider
```md
- [[step 1]]
- [[...]]
```
or something along those lines instead, for consistent syntax (`[[]]`) with our other comment blocks in the templates.